src/appointment/appointment.dto.ts

```diff
@@ -0,0 +1,42 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { prop } from '@typegoose/typegoose';
+
+export class AppointmentDTO {
```

Comment:
Perhaps this `AppointmentDTO` should be rename to `CreateAppointmentDTO` to prevent confusion with `EditAppointmentDTO`. Let's ask for @UtopiaBeam opinion too.

Reply:
I agree with @theoatboy139, `@prop` decorators should be removed too.